docker/local-dev-full-compose.yml

```diff
@@ -42,7 +42,35 @@ services:
         "--provider-type=msp",
         "--max-storage-capacity=4294967295",
         "--jump-capacity=1073741824",
-        "--name=sh-msp",
+        "--name=sh-msp-1",
+        "--no-hardware-benchmarks",
+        "--unsafe-rpc-external",
+        "--rpc-methods=unsafe",
+        "--port=30350",
+        "--rpc-cors=all",
+        "--node-key=${NODE_KEY}",
```

Comment:
Does this mean that they will both have the same `NODE_KEY`? Shouldn't they have different ones?

Reply:
No, on `test/util/fullNet/helpers.ts`, when containers are started, these are passed their respective `NODE_KEY`. Like so:

```
await compose.upOne("sh-msp-1", {
      cwd: cwd,
      configAsString: updatedCompose,
      log: true,
      env: {
        ...process.env,
        NODE_KEY: ShConsts.NODE_INFOS.msp1.nodeKey,
        BSP_IP: bspIp,
        BSP_PEER_ID: bspPeerId,
        MSP_ID: ShConsts.DUMMY_MSP_ID
      }
    });
```